src/server/game/Entities/Player/PlayerUpdates.cpp

```diff
@@ -0,0 +1,2048 @@
+#include "BattlefieldMgr.h"
+#include "Channel.h"
+#include "ChannelMgr.h"
+#include "Formulas.h"
+#include "GridNotifiers.h"
+#include "Group.h"
+#include "Guild.h"
+#include "InstanceScript.h"
+#include "Language.h"
+#include "OutdoorPvPMgr.h"
+#include "Pet.h"
+#include "Player.h"
+#include "SavingSystem.h"
+#include "ScriptMgr.h"
+#include "SkillDiscovery.h"
+#include "SpellAuraEffects.h"
+#include "UpdateFieldFlags.h"
+#include "Vehicle.h"
+#include "WeatherMgr.h"
+
+// Zone Interval should be 1 second
+#define ZONE_UPDATE_INTERVAL (1 * IN_MILLISECONDS)
+
+#define CINEMATIC_UPDATEDIFF 500
+#define CINEMATIC_LOOKAHEAD (2 * IN_MILLISECONDS)
+
+void Player::Update(uint32 p_time) {
```

Comment:
the style is changed... the braces should be on new line to make it consistent with the rest of the proejct

Reply:
Yeah this is why i marked it as WIP, please merge this after the clang format PR has been merged so both files are in the proper format